libcontainer/rootfs_linux.go

```diff
@@ -383,6 +384,11 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string, enableCgroupns b
 				return err
 			}
 		}
+		if isRo {
+			if err := remount(m, rootfs); err != nil {
```

Comment:
also makes sense to add a comment, something like
```go
// initially mounted rw in mountPropagate, remount to ro
```
should be sufficient I guess.

Reply:
done